packaging/Dockerfile.centos8-nmstate-dev

```diff
@@ -32,6 +32,16 @@ RUN dnf -y install --setopt=install_weak_deps=False \
                    git \
                    iproute \
                    rpm-build \
+                   # Below packages for pip (used by tox) to build
```

Comment:
This should not be needed. The RPMs for pygobject/dbus-python should be used.

Reply:
Done thanks!

Reply:
If I do not install these packages I am getting a dependency error. Please, look at this log:

https://travis-ci.com/nmstate/nmstate/jobs/282422509

What do you think? Thanks!

Reply:
Ok, for CentOS as long as we test only the system python version, setting `sitepackages=true` in tox.ini should probably fix it. But maybe we need to have this...
A cleaner command would be to use `yum builddep python-gobject dbus-python` (not sure about the actual package names) but I see now that we do something similar in the Fedora container spec. Therefore this is maybe ok.

Reply:
I will try to fix this. Thanks!